ygen/codegen.go

```diff
@@ -291,7 +294,7 @@ func mappedDefinitions(yangFiles, includePaths []string, opts IROptions) (*mappe
 // mapped with path compression enabled. The set of modules that the current code generation
 // is processing is specified by the modules slice. This function returns slice of errors
 // encountered during processing.
-func findMappableEntities(e *yang.Entry, dirs map[string]*yang.Entry, enums map[string]*yang.Entry, excludeModules []string, compressPaths bool, modules []*yang.Entry) util.Errors {
+func findMappableEntities(e *yang.Entry, dirs map[string]*yang.Entry, enums map[string]*yang.Entry, excludeModules []string, compressPaths, ignoreUnsupportedStatements bool, modules []*yang.Entry) util.Errors {
```

Comment:
Can we add a unit test for `findMappableEntities` that covers this case please?

Reply:
Done.